scripts/cpu_percentage.sh

```diff
@@ -28,6 +28,11 @@ print_cpu_percentage() {
       usage="$(cached_eval WMIC cpu get LoadPercentage | grep -Eo '^[0-9]+')"
       # shellcheck disable=SC2059
       printf "$cpu_percentage_format" "$usage"
+    else
+    if is_osx; then
+      load=$(cached_eval ps aux | awk '{print $3}' | tail -n+2 | awk '{s+=$1} END {print s}')
+      cpus=$(cpus_number)
+      echo "$load $cpus" | awk -v format="$cpu_percentage_format" '{printf format, $1/$2}'
     else
       load=$(cached_eval ps -aux | awk '{print $3}' | tail -n+2 | awk '{s+=$1} END {print s}')
```

Comment:
I think actually we can drop the `-`

```suggestion
    else
      load=$(cached_eval ps aux | awk '{print $3}' | tail -n+2 | awk '{s+=$1} END {print s}')
```